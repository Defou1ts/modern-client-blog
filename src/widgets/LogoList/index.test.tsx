import React from 'react';

import renderer from 'react-test-renderer';

import { LogoList } from './index';

describe('LogoList', () => {
	it('LogoList snapshot', () => {
		const component = renderer.create(<LogoList />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
