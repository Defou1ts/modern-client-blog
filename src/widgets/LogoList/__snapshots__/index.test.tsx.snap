// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoList LogoList snapshot 1`] = `
<article
  className="wrapper"
>
  <div
    className="info"
  >
    <p
      className="p small subtitle"
    >
      home.logo-list-subtitle
    </p>
    <h4
      className="h h4 title"
    >
      home.logo-list-title
    </h4>
  </div>
  <div
    className="logolist"
  >
    <img
      alt="logo 1"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      title="logo 1"
      width={40}
    />
    <img
      alt="logo 2"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      title="logo 2"
      width={40}
    />
    <img
      alt="logo 3"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      title="logo 3"
      width={40}
    />
    <img
      alt="logo 4"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      title="logo 4"
      width={40}
    />
    <img
      alt="logo 5"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      title="logo 5"
      width={40}
    />
  </div>
</article>
`;
